contrib/Dockerfile

```diff
@@ -0,0 +1,16 @@
+ARG label
+
+FROM readthedocs/build:${label}
+
+ARG uid
```

Comment:
For some reason, docker was not picking these up if they were at the top of the file? :man_shrugging:

Reply:
It's in the docs 

> FROM instructions support variables that are declared by any ARG instructions that occur before the first FROM.